whylog/config/__init__.py

```diff
@@ -119,14 +138,26 @@ def _load_parsers(self):
         )
 
     def _load_rules(self):
-        return [
-            RegexRuleFactory.from_dao(serialized_rule, self._parsers)
-            for serialized_rule in self._load_file_with_config(self._rules_path)
-        ]
+        loaded_rules = defaultdict(list)
+        for serialized_rule in self._load_file_with_config(self._rules_path):
+            loaded_rule = RegexRuleFactory.from_dao(serialized_rule, self._parsers)
+            loaded_rules[serialized_rule["effect"]].append(loaded_rule)
+        return loaded_rules
 
     def _load_log_types(self):
-        #TODO fix log types loading
-        pass
+        matchers = defaultdict(list)
+        matcher_definitions = self._load_file_with_config(self._log_type_path)
+        for definition in matcher_definitions:
+            if definition['matcher_class_name'] == 'RegexFilenameMatcher':
+                matcher = RegexFilenameMatcherFactory.from_dao(definition)
+                matchers[definition['log_type_name']].append(matcher)
+            else:
+                # TODO: place for other filename matchers
+                pass
```

Comment:
Exception should be raised (eg. `NotImplementedError`). We should never fail quietly.